Makefile

```diff
@@ -0,0 +1,29 @@
+all:
+	make clean
+	make compile
+
+everything:
+	make clean
+	make compile
+	make upload
+	make uninstall
+	make install
+	make install
```

Comment:
Whatever I did, pip didnt install the latest version on first run,
running install twice installed the latest version. 

Intentionally installing twice for that reason.